pkg/helm_actions/upgrade_release.go

```diff
@@ -0,0 +1,32 @@
+package helm_actions
+
+import (
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+)
+
+func UpgradeRelease(ns, name, url string, conf *action.Configuration) (interface{}, error) {
+	cmd := action.NewInstall(conf)
+	cmd.Namespace = ns
```

Comment:
How about doing a 
```
// NewInstall creates a new Install object with the given configuration.
func NewInstall(cfg *Configuration, namespace string) *Install {
	return &Install{
		cfg: cfg,
                 namespace: namespace,
	}
}
```
and then use that in 
```
cmd := action.NewInstall(conf,ns)
``` 
?

Reply:
namespace will now be set upfront in conf itself